src/open_inwoner/templates/pages/cases/statuses.html

```diff
@@ -0,0 +1,81 @@
+
+            {% if forloop.last %}
+                {# Current active status class #}
+                <li class="status-list__list-item {% if case.end_statustype_data %}status--current status-list__list-item--{{ status.status_indicator }} status--open{% endif %} {# Final status class #}{% if not case.end_statustype_data %}status--final status-list__list-item--info status--open{% endif %}">
+                {% firstof case.id|slugify as id %}
+                {% firstof 'content-'|add:id|add:'-id' as content_id %}
+                    {% icon icon="circle" extra_classes="status-step" %}
+                    <div class="status-list__notification status-list__notification--{{ status.status_indicator }}">
+                        <button class="button--borderless {% if case.end_statustype_data %}status-list__button{% endif %} {# Final status class #}{% if not case.end_statustype_data %}status-list__open-button{% endif %}" aria-controls="{{ content_id }}" aria-expanded="true" id="{{ id }}">
+                            <span class="link link--bold">{{ status.label }}</span>
+                            {% if case.end_statustype_data %}{% icon icon="expand_more" icon_position="after" outlined=True %}{% endif %} </button>
+                        <div class="status-list__notification-content {% if case.end_statustype_data %}status-content--open{% endif %} {% if not case.end_statustype_data %}status-content--open{% endif %}" id="{{ content_id }}">
+                            <p class="p p-text p--compact">
+                                {# Should be ZaakTypeResultaatTypeConfig configurable description #}
+                                {# {{ status.statustype.omschrijving }} #}
+                                {{ status.status_indicator_text }}</p>
+                            <p class="p p-date p--compact">{{ status.date|date }}</p>
+                            <p class="p">
+                            {% if case.internal_upload_enabled or case.external_upload_enabled %}
+                                {% button href="#external-upload" text=_("Scroll naar document uploaden") title=_("Ga direct naar document upload sectie.") primary=True icon="arrow_downward" icon_position="after" %}
+                            {% endif %}
+                            {# IF there is a CTA button in the final status, show this #}
+{#                                {% with button_url=case.end_statustype_data.call_to_action_url %}#}
+{#                                    {% if button_url %}#}
+{#                                        {% translate case.end_statustype_data.call_to_action_text as button_text %}#}
+{#                                            {% button href=button_url text=button_text title=_("Open de bijlage") primary=True icon="arrow_forward" icon_position="after" %}#}
+{#                                    {% endif %}#}
+{#                                {% endwith %}#}
+                            {# ENDIF #}
```

Comment:
This functionality will be added by PR 829, so it's not necessary to include the commented out version here.